webpack.config.js

```diff
@@ -56,7 +62,7 @@ module.exports = {
       {test: /\.ts$/, loaders: ['ts-loader'], exclude: /(node_modules|libs)/},
       {test: /\.html$/, loader: 'raw', exclude: /(node_modules|libs|dist|tsd|bower)/},
       // stylesheets
-      {test: /\.less$/, exclude: /(node_modules|lib)/, loader: ExtractTextPlugin.extract('style-loader', 'css-loader')},
+      {test: /\.less$/, exclude: /(node_modules|lib)/, loader: ExtractTextPlugin.extract('style-loader', 'css-loader!less-loader')},
```

Comment:
I think we should probably switch over to Sass instead of LESS since we are now working with a Rails environment.